core/tests/protractor_utils/AdminPage.js

```diff
@@ -152,6 +159,37 @@ var AdminPage = function() {
       'Could not set role successfully');
     return true;
   };
+
+  this.viewRolesbyRole = function(role) {
```

Comment:
This function name doesn't make sense to me, could you explain?

Reply:
I think this name suits the thing that the code intends to do...

![Screenshot 2019-10-22 at 1 06 56 AM](https://user-images.githubusercontent.com/30312043/67237298-9e3bb500-f468-11e9-8de1-8012862ddd3c.png)

Reply:
Explained below..

Reply:
Maybe this method should return a list of the username of users assigned to a specific role? If so I think you can change the method name to getUsersAsssignedToRole (or something around that)?

Reply:
Ok done